blog-content/articles/save-your-future-self-by-encapsulating-test-input-preparation.md

```diff
@@ -0,0 +1,97 @@
+---
+date: 2020-01-07
+summary: Examples of how to reduce boilerplate and brittleness of your test input preparation 
+---
+
+# Tests Brittleness and Verbosity slows you down
+
+In [Testing Makes You Faster Day one](entry/testing-makes-you-faster-day-one) article I claimed that writing tests makes you faster.
+However there are test patterns that most likely will slow you down and you need to be ready to handle them. 
+
+Writing tests for a simple logic with a simple input is a breathe. Take for example a test for a simple calculator class. 
+Two numeric inputs and we are done: 
+
+:include-file: com/example/portfolio/SimpleCalculatorTest.java {title: "SimpleCalculatorTest.java"}
+
+Unfortunately real apps require real domain, and real domain is rarely represented by a couple of numbers.
```

Comment:
I feel like this paragraph is incomplete - I think you are saying the following:
1. real apps need a real domain
2. this example is a false domain
3. the false domain is insufficient because it is too shallow
4. So let's take a better example with a domain deep enough to write meaningful tests for.

I feel like #3 is implied, and #4 is omitted. 

This is not essential, but it stood out at me.